<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReserveWindow</class>
 <widget class="QMainWindow" name="ReserveWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ReserveWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="connectToServerLineEdit">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>430</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>127.0.0.1:1221</string>
    </property>
   </widget>
   <widget class="QPushButton" name="connectToMainServerButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>430</y>
      <width>171</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect to Main Server</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="logWidget">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>20</y>
      <width>371</width>
      <height>521</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="disconnectButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>510</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Disconnect</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="connectionTable">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>361</width>
      <height>311</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Client ID</string>
     </property>
    </column>
   </widget>
   <widget class="QLineEdit" name="connectionsCountLineEdit">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>360</y>
      <width>181</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>370</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Connections count</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
